test/extended/jenkins/plugin.go

```diff
@@ -203,9 +203,11 @@ var _ = g.Describe("[jenkins][Slow] openshift pipeline plugin", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			g.By("get build console logs and see if succeeded")
-			err = waitForJenkinsActivity(fmt.Sprintf("http://%s/job/test-plugin-job/1/console", hostPort), password, "Finished: SUCCESS", 200)
+			err = waitForJenkinsActivity(fmt.Sprintf("http://%s/job/test-plugin-job/1/console", hostPort), "GET", password, "Finished: SUCCESS", 200)
 			o.Expect(err).NotTo(o.HaveOccurred())
 
+			g.By("trigger test connection logic, check for success")
+			immediateInteractionWithJenkins(fmt.Sprintf("http://1.3.4.5/job/test-plugin-job/descriptorByName/com.openshift.jenkins.plugins.pipeline.OpenShiftScaler/testConnection", "POST", hostPort), "POST", password, fmt.Fprint(body, "apiURL=&authToken="), 200)
```

Comment:
Whoops, I was so intent on not missing any uses, I forgot to change mine. It's correct now!